### PR TITLE
Proposal to extend directions

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9970

@dstrome what do you think?

### CODE REVIEW

File: Teams/devices/remote-sign-in-and-sign-out.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ To sign in a device that has been signed out, go to the corresponding device pag
 
 1. Select the device you want to sign in.
 
-2. Go to the **Actions** menu, and select **Sign in a user**.
+2. Go to the **Actions** menu at the top right hand side of the page, and select **Sign in a user**.
```
Comment (JohanFreelancer9) [inline Teams/devices/remote-sign-in-and-sign-out.md:42]:
```suggestion
2. Go to the **Actions** menu at the top right-hand side of the page, and select **Sign in a user**.
```

File: Teams/devices/remote-sign-in-and-sign-out.md [hunk 2]
```diff
@@ -39,13 +39,13 @@ To sign in a device that has been signed out, go to the corresponding device pag
...
 
 ## Remote sign out
 
 1. Select the device you want to sign out from the corresponding page.
 
-2. Go to the **Actions** menu, and select **Sign out**.
+2. Go to the **Actions** menu at the top right hand side of the page, and select **Sign out**.
```
Comment (JohanFreelancer9) [inline Teams/devices/remote-sign-in-and-sign-out.md:48]:
```suggestion
2. Go to the **Actions** menu at the top right-hand side of the page, and select **Sign out**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
Thank you for the suggestion, however we avoid getting that specific as UI element placement can change over time.

**Comment by get-itips** [comment]
@dstrome I agree with that.

**Pull request closed without merging**